src/lib/modules/Document/DocumentListEntry/DocumentListEntry.js

```diff
@@ -20,6 +20,22 @@ class DocumentListEntry extends Component {
     this.metadata = props.metadata;
   }
 
+  renderEItemTags = (eitems) => {
```

Comment:
for better understanding in semantic ui context, and to make the connection with eitem types defined above:
```suggestion
  renderEItemTypeLabels = (eitems) => {
```